cnd/src/db/tables.rs

```diff
@@ -0,0 +1,733 @@
+use crate::{
+    db::{
+        schema::{
+            address_hints, halight_swap_details, herc20_swap_details, secret_hashes,
+            shared_swap_ids, swaps,
+        },
+        wrapper_types::{
+            custom_sql_types::{Text, U32},
+            BitcoinNetwork, Erc20Amount, EthereumAddress, Satoshis,
+        },
+        Error, Sqlite,
+    },
+    lightning,
+    swap_protocols::{self, rfc003, HashFunction, LocalSwapId, Role},
+};
+use diesel::{self, prelude::*, RunQueryDsl};
+use libp2p::{Multiaddr, PeerId};
+
+// TODO: Audit 'pub' scope on all structs.
+
+#[derive(Identifiable, Queryable, PartialEq, Debug)]
+#[table_name = "swaps"]
```

Comment:
Sometimes singular, sometimes plural. I think the spike defined singular :)

Reply:
@da-kami and I debated this, diesel works by default with plural so we went for plural.  (The drop statements where a left over of this debate that I missed.)

Reply:
Really, diesel cares about this? Wow :D

Reply:
I wouldn't go so far as saying it 'cares', there are ways to make it work but it is 'opinionated' and naming it like this makes the default marco behaviours work.

Reply:
Good enough reason for me to go with diesel's opinion then, convention over configuration :)